pkgs/applications/audio/bitwig-studio/bitwig-studio5.nix

```diff
@@ -123,17 +121,17 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  meta = with lib; {
+  meta = {
```

Comment:
(I am pretty new in reviewing nixpkgs code, is there some reason getting rid of the `with` ?)

Reply:
`with` is considered bad practice because it pollutes the namespace.